shadows/framework/src/main/java/org/robolectric/shadows/ShadowMediaCodec.java

```diff
@@ -167,11 +167,11 @@ protected void native_configure(
       Object crypto,
       Object descramblerBinder,
       Object flags) {
-    configure(
+    inner_configure(
         (String[]) keys, (Object[]) values, (Surface) surface, (MediaCrypto) crypto, (int) flags);
   }
 
-  private void configure(
+  private void inner_configure(
```

Comment:
Maybe `innerConfigure` is better than `inner_configure` in Java? I think a_b naming convention is not recommend in Java.

Reply:
Fair. I'll fix it before the next force-push.